project/webapp/templates/static_templates/static_footer.html

```diff
@@ -5,11 +5,22 @@
     <!-- Top block for external links -->
     <div class="container">
         <div class="row center">
-            <div class="external-links grey-text text-darken-2">
-                <a class="grey-text text-darken-1" href="mailto:mitchell@civiwiki.org">Email</a> |
-                <a class="grey-text text-darken-1" href="https://www.facebook.com/CiviWiki/"  target="_blank">Facebook</a> |
-                <a class="grey-text text-darken-1" href="https://twitter.com/civiwiki"  target="_blank">Twitter</a> |
-                <a class="grey-text text-darken-1" href="https://github.com/CiviWiki/OpenCiviWiki"  target="_blank">GitHub</a>
+            <div class="col s6 m3 push-m6 right-align">
+                <div class="external-link-header subtitle-lato grey-text">Connect with Us</div>
+                <div class="external-links grey-text text-darken-2 right-align">
+                    <div class="external-link-item"><a class="grey-text text-darken-1" href="mailto:mitchell@civiwiki.org">Email</a></div>
```

Comment:
To improve readability, consider nesting these child elements, rather than placing them inline. 
```html
div
    a
        text
    /a
/div
```


This allows vertical scanning of the code to read it faster.